src/main/AuthenticationResponse.ts

```diff
@@ -4,19 +4,40 @@ import { JWK } from 'jose/types';
 import AuthenticationRequest from './AuthenticationRequest';
 import { createDiscoveryMetadataPayload } from './AuthenticationResponseRegistration';
 import { PresentationExchange } from './PresentationExchange';
-import { DIDJwt, DIDres, State } from './functions';
-import { signDidJwtPayload, verifyDidJWT } from './functions/DidJWT';
-import { getPublicJWKFromHexPrivateKey, getThumbprint } from './functions/Keys';
-import { JWT, SIOP, SIOPErrors } from './types';
 import {
+  getIssuerDidFromPayload,
+  getNonce,
+  getPublicJWKFromHexPrivateKey,
+  getResolver,
+  getState,
+  getThumbprint,
+  parseJWT,
+  signDidJwtPayload,
+  verifyDidJWT,
+} from './functions';
+import { validateWithDid } from './functions/LinkedDomainValidations';
```

Comment:
This should be properly exported, so we can import it from ./functions

Reply:
changed